crates/config/src/config.rs

```diff
@@ -91,13 +91,14 @@ pub struct Web3IndexerConfig {
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "lowercase")]
-pub enum TestMode {
-    Enable,
-    Disable,
+pub enum NodeMode {
+    FullNode,
+    Test,
+    ReadOnly,
 }
 
-impl Default for TestMode {
+impl Default for NodeMode {
     fn default() -> Self {
-        TestMode::Disable
+        NodeMode::FullNode
```

Comment:
```suggestion
        NodeMode::ReadOnly,
```

ReadOnly is default mode.